src/SimpleJson/SimpleJson.cs

```diff
@@ -46,7 +46,7 @@
 
 // original json parsing code from http://techblog.procurios.nl/k/618/news/view/14605/14863/How-do-I-write-my-own-parser-for-JSON.html
 
-#if NETFX_CORE
+#if NETFX_CORE || DOTNET54

```

Comment:
Though I'm not so sure this should be added at all. You should probably go ahead and add

```csharp
... || DOTNET5_1 || DOTNET5_2 || DOTNET5_3  || DOTNET5_5 || DOTNET5_6 || NETSTANDARD1_0 || NETSTANDARD1_1 || NETSTANDARD1_2 // and on and on...
```

See where I'm going? If anything, the things that should be #ifdef'ed should be behind a feature flag, not a platform.